backend/src/db/payments.js

```diff
@@ -0,0 +1,26 @@
+const models = require('../models');
+const db = require('../sequalize_pg');
+
+const createPayment = async (paymentObj) => {
+    await db.sync();
+    models.payments.create(paymentObj);
+};
+
+const getPayment = async (ordernumber) => {
+    return models.payments.find({ where: { ordernumber } });
+};
+
+const updatePaymentStatus = async (authCode, status) => {
```

Comment:
This doesn't need to be an `async` function as it doesn't need to await anything. Use regular function and return the promise as usual.